arroyo-state/src/parquet.rs

```diff
@@ -623,7 +694,14 @@ impl Default for ParquetStats {
 }
 
 impl RecordBatchBuilder {
-    fn insert(&mut self, key_hash: u64, timestamp: SystemTime, key: Vec<u8>, data: Vec<u8>) {
+    fn insert(
```

Comment:
Rather than `insert` taking a DataOperation we could have `insert()` and `remove()` separately.

Reply:
I thought about that, but what's really happening is an _insert_ into the `RecordBatchBuilder`, so in the context of this implementation of a backing store and this parquet record batch builder, I think these semantics are accurate.